frontend/src/components/RandomPlayerOrder.js

```diff
@@ -0,0 +1,95 @@
+import styled from 'styled-components';
+import ButtonBig from './ButtonBig';
+import ButtonTab from './ButtonTab';
+import { useState } from 'react';
+import SessionPlayerTile from './SessionPlayerTile';
+
+export default function RandomPlayerOrder({
+    session,
+    randomizePlayer,
+    startTimer,
+}) {
+    const [firstPlayer, setFirstPlayer] = useState(false);
+
+    const randomizePlayerOrder = () => {
+        let playerListToRandomize = [...session.playerList];
+        let randomizedPlayers = [];
+        while (playerListToRandomize.length > 0) {
+            let randomIndex = Math.floor(
```

Comment:
```suggestion
        const playerListToRandomize = [...session.playerList];
        const randomizedPlayers = [];
        while (playerListToRandomize.length > 0) {
            const randomIndex = Math.floor(
```

Good thinking on the algorithm! 

Good to know: there is a very easy way to randomize an array using sort
```
const playerListToRandomize = [...session.playerList];
playerListToRandomize.sort(() => 0.5 - Math.random())
```

This looks a bit hacky but I've seen it so often that I think it is the idiomatic way to do it in js.

Reply:
goo to know. I will try it in a future version ^^